pkgs/top-level/all-packages.nix

```
@@ -2540,6 +2540,8 @@ in
 
   gif-for-cli = callPackage ../tools/misc/gif-for-cli { };
 
+  giph = callPackage ../applications/video/giph/default.nix { };
```

```suggestion
  giph = callPackage ../applications/video/giph { };
```
`default.nix` is inferred when specifying a directory.